dashboard/app/helpers/level_starter_assets_helper.rb

```diff
@@ -1,6 +1,23 @@
 module LevelStarterAssetsHelper
   S3_BUCKET = 'cdo-v3-assets'.freeze
   S3_PREFIX = 'starter_assets/'.freeze
+  MAX_RESIZE_SIZE = 20_000_000 # 20 MB
+  MAX_DIMENSION_PIXELS = 2048
+
+  def self.try_resize_file(file, extension)
```

Comment:
Should the target size be an (optional) argument to this function so we can specify 2048 specifically for AI Chat?